packages/webbundle-webpack-plugin/README.md

```diff
@@ -69,24 +69,35 @@ for more details.
 /* webpack.config.js */
 const path = require('path');
 const WebBundlePlugin = require('webbundle-webpack-plugin');
-const { WebBundleId, parsePemKey } = require('wbn-sign');
+const {
+  NodeCryptoSigningStrategy,
+  parsePemKey,
+  readPassphrase,
+  WebBundleId,
+} = require('wbn-sign');
 require('dotenv').config({ path: './.env' });
 
-module.exports = () => {
-  const key = parsePemKey(process.env.ED25519KEY);
+module.exports = async () => {
+  const key = parsePemKey(process.env.ENC_ED25519KEY, await readPassphrase());
 
   return {
     entry: './src/index.js',
-    output: {
-      path: path.resolve(__dirname, 'dist'),
-      filename: 'app.js',
-    },
+    output: { path: path.resolve(__dirname, 'dist'), filename: 'app.js' },
     plugins: [
       new WebBundlePlugin({
         baseURL: new WebBundleId(key).serializeWithIsolatedWebAppOrigin(),
         static: { dir: path.resolve(__dirname, 'static') },
-        output: 'signed.wbn',
-        integrityBlockSign: { key },
+        output: 'signed.swbn',
+        integrityBlockSign: {
+          strategy: new NodeCryptoSigningStrategy(key),
+        },
+        headerOverride: {
+          'cross-origin-embedder-policy': 'require-corp',
+          'cross-origin-opener-policy': 'same-origin',
+          'cross-origin-resource-policy': 'same-origin',
+          'content-security-policy':
+            "base-uri 'none'; default-src 'self'; object-src 'none'; frame-src 'self' https:; connect-src 'self' https:; script-src 'self' 'wasm-unsafe-eval'; img-src 'self' https: blob: data:; media-src 'self' https: blob: data:; font-src 'self' blob: data:; require-trusted-types-for 'script';",
```

Comment:
Could you update this to the latest CSP values from here https://github.com/WICG/isolated-web-apps? (mainly `frame-src` has been changed, I think).